docs/core/diagnostics/built-in-metrics-aspnetcore.md

```diff
@@ -67,6 +67,8 @@ The time ends when:
 - All response data has been sent.
 - The context data structures for the request are being disposed.
 
+When using OpenTelemetry, the default buckets for this metric are set to { 0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 }.
```

Comment:
OpenTelemetry SDK would be updated to remove the bucket with the value `0`. This bucket is not useful as we wouldn't record negative values. You would have to update the documentation again then.

https://github.com/open-telemetry/semantic-conventions/issues/298

Reply:
done